examples/complete/atmos.yaml

```diff
@@ -128,3 +128,64 @@ commands:
         steps:
           - echo Playing ping-pong...
           - echo pong
+
+# Integrations
+integrations:
+
+  # Atlantis integration
+  # https://www.runatlantis.io/docs/repo-level-atlantis-yaml.html
+  atlantis:
+    # Path and name of the Atlantis config file `atlantis.yaml`
+    # Supports absolute and relative paths
+    # All the intermediate folders will be created automatically (e.g. `path: /config/atlantis/atlantis.yaml`)
+    # Can be overridden on the command line by using `--output-path` command-line argument in `atmos atlantis generate repo-config` command
+    # If not specified (set to an empty string/omitted here, and set to an empty string on the command line), the content of the file will be dumped to `stdout`
+    path: "atlantis.yaml"
+
+    # Config templates
+    # Select a template by using the `--config-template <config_template>` command-line argument in `atmos atlantis generate repo-config` command
+    config_templates:
+      config-template-1:
+        version: 3
+        automerge: true
+        delete_source_branch_on_merge: true
+        parallel_plan: true
+        parallel_apply: true
+        allowed_regexp_prefixes:
```

Comment:
Can any key be added here and show up in the config file ? or does it need to match a specific list of keys ?

Reply:
Same question for `project_templates` and `workflow_templates` keys

Reply:
Answered here https://github.com/cloudposse/atmos/pull/189#discussion_r964226083